src/scss/grommet-core/_objects.accordion.scss

```diff
@@ -0,0 +1,10 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+.grommet .#{$grommet-namespace}accordion {
```

Comment:
This is a specificity problem. 

In hpe.min.css:
```
.grommet p {                                    // specificity level 11
  margin-top: 24px;
  margin-bottom: 24px;
}
```
Which would be more specific than the accordion styles i'm adding.
Anyway I'll remove the custom margin top & bottom & `.grommet` from the styles.